nixos/modules/services/misc/sdrplay.nix

```diff
@@ -0,0 +1,15 @@
+{ config, lib, pkgs, sdrplay, ... }:
+with lib;
+{
+  options.services.sdrplay_api = {
+    enable = mkEnableOption "SDRPlay API service";
+  };
+
+  config = mkIf config.services.sdrplay_api.enable {
+    systemd.services.sdrplay_api = {
+      wantedBy = [ "multi-user.target" ];
+      serviceConfig.ExecStart = "${sdrplay}/bin/sdrplay_apiService";
```

Comment:
```suggestion
      serviceConfig.ExecStart = "${pkgs.sdrplay}/bin/sdrplay_apiServipkgs
```

Reply:
I guess the trailing `pkgs` instead of a `"` in your line was accidental?
I've applied the change of accessing `sdrplay` through `pkgs.sdrplay` and removed the function argument at the top.